best-practices/client-side/framework/react/README.md

```diff
@@ -1,3 +1,16 @@
-# react
+# React
 
-## all info on current dev with react, how to spin up a boilerplate
\ No newline at end of file
+## Intro
+
+-   [React.js](https://reactjs.org/docs/getting-started.html) often referred to as React or ReactJS is an open-source JavaScript library for building user interfaces or UI components, created by Facebook.
+
+-   [React Native](https://reactnative.dev/) is an open-source mobile application framework created by Facebook to develop applications for Android, iOS, Web and UWP by enabling developers to use React along with native platform capabilities.
+
+## Spining up a boilerplate
+
+you can refer to [Aerostar](https://github.com/Shift3/Aerostar) and [sundial-mobile-app](https://github.com/Shift3/sundial-mobile-app) repositories.
```

Comment:
I don't believe these two projects are good boilerplate projects. 

I would expect a good boilerplate project to be:
1. Can be reused over and over without change
2. Easy to set up
3. Contain all common and documented patterns
4. Contains some testing (in an ideal world, code coverage would be near 70% or higher)
5. Since react is client facing, the projects should be server agnostic (can be used with any server)

Both projects are in a state of active development at the moment and will most likely change to meet the clients specifications. I am concerned developers looking to spin up a react project would fork them without understanding why we made the assumptions and trade-offs in relation to the project specifications. They are also both highly coupled to their respective server REST APIs.

They are, however, great examples of non-trivial projects. The Standards and Practices repo is public, so anyone wanting to see Shift3s development practices should be able to see these issues/PRs/READMEs. Unfortunately, those links are private to the organization, so if we wish to include them, you may want to add some verbiage to the effect of "if you would like access to these example projects, please reach out and request access".

@Fleury14, @kmausolf, and/or @codysmith287 would https://github.com/Shift3/rn-by-example be a better example of a boilerplate project?

Reply:
Hmmmmm, I think as far as reusability, yes, rn-by-example fits the bill. For the React Native portion, I would definitely reference that.

As far a React web goes [Project Status/Batsignal](https://github.com/Shift3/bw-project-status) would probably be the first place I'd point someone as a recent React project that has both React and React Native. I wouldn't reference Listing Alert because I wrote that prior to Hooks being a thing. But both these examples are less of a "How to spin up a project" and more of a "What a project should somewhat look like".

Ultimately, I think the best answer becomes an actual boiler plate similar to the angular boiler plate, but we don't have one of those for React Web (yet).